### PR TITLE
TRITON-2344 Agents should include the build stamp in the image version string

### PR DESCRIPTION
**Pull request by bahamat**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -102,10 +103,11 @@ release: all docs $(SMF_MANIFESTS)
     $(TOP)/lib \
     $(TOP)/node_modules \
     $(TOP)/npm \
-    $(TOP)/package.json \
```
Comment (danmcd) [inline Makefile:105]: Same question as other reviews...

**Review by danmcd — COMMENTED** [review]
Why removing package.json?  I'm missing something but I'll be damned if I know what.

File: Makefile [hunk 2]
```diff
@@ -102,10 +103,11 @@ release: all docs $(SMF_MANIFESTS)
...
     $(TOP)/sbin \
     $(TOP)/smf \
     $(DSTDIR)
+	json -f $(TOP)/package.json -e 'this.version += "-$(STAMP)"' \
```
Comment (danmcd) [inline Makefile:109]: Shouldn't you have alterations in place for package.json as well?

**Review by danmcd — COMMENTED** [review]
Took me a bit to understand how manifest.tmpl was used, but I'm not sure you shouldn't ALSO be changing package.json.

File: Makefile (hunk 2 above)
Comment (danmcd) [reply Makefile:109]: Nope, you shouldn't.  Missing the += bit.

**Review by danmcd — APPROVED** [review]

**Review by travispaul — APPROVED** [review]

**Pull request merged**